doc/classes/XRInterface.xml

```diff
@@ -32,6 +32,13 @@
 				Returns the name of this interface (OpenVR, OpenHMD, ARKit, etc).
 			</description>
 		</method>
+		<method name="get_play_area" qualifiers="const">
+			<return type="PackedVector3Array">
+			</return>
+			<description>
+				Returns an array of vectors that denotes the physical play area mapped to the virtual space around the [XROrigin3D] point. This returns an empty array if this feature is not supported or if the information is not yet available.
```

Comment:
Maybe you should clarify what the array of vectors represents exactly. Are those ordered points that can be linked by segments to create a closed shape?

Reply:
That may be tricky to define given each platform may use a different format. 

@BastiaanOlij We should start by documenting what is currently used now, and see if it's possible to have a common format for the currently supported platforms.

Reply:
In addition, Oculus introduced `desktop` and `couch` guardian mode. So it may be useful to have a `get_play_area_qualifier` method to allow developers to differentiate between each mode.
For example, on PC the modes would mostly default to `roomscale`/`stationary`.

Reply:
@akien-mga yeah it should make a convex polygon that you could extrude upwards though generally speaking you wouldn't use this to visualise the boundaries as the XR runtime usually already does that. 

It's more for when you want to introduce gameplay elements or provide guidance to users where you need to know how much physical movement is possible.

@m4gr3d for the shape of the play area so far both Oculus desktop, OpenVR and OpenXR seem to provide a rectangle that fits within the guardian space that was setup. I don't know what the Quest offers here.

As for changing/querying the mode, that is something I think we should add as well but where we do need to do more investigation because there seems to be a wild growth of options here and our own `center_to_hmd` feature overrides this as well.

Reply:
@m4gr3d this is a nice page to get a quick overview: https://developer.oculus.com/documentation/native/android/mobile-guardian/

It's interesting to see what Oculus has done here and it makes sense when you think of how well the Quest mapping of a room works. You could mix real world elements into your place space but doing so will mean you shouldn't recenter your position as this real world to virtual world mapping breaks. They solved it simply by having two roomscale (stage) modes. 

The same I think will apply to Hololens support at some point.

I'm going to add our own enum for different modes that we can then map to the correct mode for whatever plugin is available.